.github/workflows/lint.yml

```diff
@@ -0,0 +1,107 @@
+name: Lint
+
+on:
+  pull_request:
+
+jobs:
+  # Determine what the js changes are
+  js-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      packages: ${{ steps.filter.outputs.changes }}
+    steps:
+      # Use paths-filter to determine which paths have changes
+      # https://github.com/dorny/paths-filter
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          # Format of key: value. When there is a change detected in value, the
+          # key will be returned in the output.
+          filters: |
+            packages/dashboard: packages/dashboard/**
+            packages/dashboard-v2: packages/dashboard-v2/**
+            packages/dnslink-api: packages/dnslink-api/**
+            packages/handshake-api: packages/handshake-api/**
+            packages/health-check: packages/health-check/**
+            packages/website: packages/website/**
```

Comment:
Is there anything in `packages` that we don't want to check? Would it make sense to have the lint run on `packages/**`, so when we add something we don't need to update this, too?

Reply:
that is a question for @kwypchlo 
I simply copied this from the previous linting.
I assume the intent here was to speed up the CI by only running the linting/testing when files were touched.

I would be fine for simplicity to just always run all the linters. It doesn't take long and protects against not updating this in the future. 

Although this lint is specifically prettier and eslint so only really applies to JS. 

The other consideration is that the dashboard packages are the only ones that perform the eslint.